second_task/src/Bow.ts

```diff
@@ -1 +1,19 @@
-// your code goes here
\ No newline at end of file
+import { Weapon } from './Weapon';
+
+export class Bow extends Weapon {
+  constructor(name:string, baseDamage: number, baseDurability: number, value: number, weight:number) {
+    super(name, baseDamage, baseDurability, value, weight)
+    this.name = name;
```

Comment:
@dianalemen, you don't need to assign variable in the derived class if you already pass them to the parent.